contrib/sbase/Makefile

```diff
@@ -0,0 +1,312 @@
+CFLAGS.sbase/mkfifo.c += -Dmain=mkfifo_main
+CFLAGS.sbase/mktemp.c += -Dmain=mktemp_main
+CFLAGS.sbase/mv.c += -Dmain=mv_main
+CFLAGS.sbase/nice.c += -Dmain=nice_main
+CFLAGS.sbase/nl.c += -Dmain=nl_main
+CFLAGS.sbase/nohup.c += -Dmain=nohup_main
+CFLAGS.sbase/printenv.c += -Dmain=printenv_main
+CFLAGS.sbase/pwd.c += -Dmain=pwd_main
+CFLAGS.sbase/readlink.c += -Dmain=readlink_main
+CFLAGS.sbase/renice.c += -Dmain=renice_main
+CFLAGS.sbase/rev.c += -Dmain=rev_main
+CFLAGS.sbase/rm.c += -Dmain=rm_main
+CFLAGS.sbase/rmdir.c += -Dmain=rmdir_main
+CFLAGS.sbase/seq.c += -Dmain=seq_main
+CFLAGS.sbase/setsid.c += -Dmain=setsid_main
+CFLAGS.sbase/sha1sum.c += -Dmain=sha1sum_main
+CFLAGS.sbase/sha224sum.c += -Dmain=sha224sum_main
+CFLAGS.sbase/sha256sum.c += -Dmain=sha256sum_main
+CFLAGS.sbase/sha384sum.c += -Dmain=sha384sum_main
+CFLAGS.sbase/sha512sum.c += -Dmain=sha512sum_main
+CFLAGS.sbase/sha512-224sum.c += -Dmain=sha512_224sum_main
+CFLAGS.sbase/sha512-256sum.c += -Dmain=sha512_256sum_main
+CFLAGS.sbase/sleep.c += -Dmain=sleep_main
+CFLAGS.sbase/split.c += -Dmain=split_main
+CFLAGS.sbase/sponge.c += -Dmain=sponge_main
+CFLAGS.sbase/strings.c += -Dmain=strings_main
+CFLAGS.sbase/sync.c += -Dmain=sync_main
+CFLAGS.sbase/test.c += -Dmain=test_main
+CFLAGS.sbase/tr.c += -Dmain=tr_main
+CFLAGS.sbase/true.c += -Dmain=true_main
+CFLAGS.sbase/tsort.c += -Dmain=tsort_main
+CFLAGS.sbase/tty.c += -Dmain=tty_main
+CFLAGS.sbase/unexpand.c += -Dmain=unexpand_main
+CFLAGS.sbase/uniq.c += -Dmain=uniq_main
+CFLAGS.sbase/unlink.c += -Dmain=unlink_main
+CFLAGS.sbase/uudecode.c += -Dmain=uudecode_main
+CFLAGS.sbase/uuencode.c += -Dmain=uuencode_main
+CFLAGS.sbase/wc.c += -Dmain=wc_main
+CFLAGS.sbase/which.c += -Dmain=which_main
+CFLAGS.sbase/whoami.c += -Dmain=whoami_main
+CFLAGS.sbase/xinstall.c += -Dmain=xinstall_main
+CFLAGS.sbase/yes.c += -Dmain=yes_main
+CFLAGS.sbase/cal.c += -Dmain=cal_main
+CFLAGS.sbase/cols.c += -Dmain=cols_main
+CFLAGS.sbase/cksum.c += -Dmain=cksum_main
+CFLAGS.sbase/find.c += -Dmain=find_main
+CFLAGS.sbase/grep.c += -Dmain=grep_main
+CFLAGS.sbase/join.c += -Dmain=join_main
+CFLAGS.sbase/kill.c += -Dmain=kill_main
+CFLAGS.sbase/ls.c += -Dmain=ls_main
+CFLAGS.sbase/od.c += -Dmain=od_main
+CFLAGS.sbase/paste.c += -Dmain=paste_main
+CFLAGS.sbase/pathchk.c += -Dmain=pathchk_main
+CFLAGS.sbase/printf.c += -Dmain=printf_main
+CFLAGS.sbase/sed.c += -Dmain=sed_main
+CFLAGS.sbase/sort.c += -Dmain=sort_main
+CFLAGS.sbase/tail.c += -Dmain=tail_main
+CFLAGS.sbase/tar.c += -Dmain=tar_main
+CFLAGS.sbase/tee.c += -Dmain=tee_main
+
+SOURCES_SBASE = $(addprefix sbase/,$(BIN:=.c))
+SOURCES_LIBUTIL = $(addprefix sbase/,$(LIBUTILSRC))
+SOURCES_LIBUTF = $(addprefix sbase/,$(LIBUTFSRC))
+OBJ_LIBUTIL = $(SOURCES_LIBUTIL:%.c=$(BUILDDIR)%.o)
+OBJ_LIBUTF = $(SOURCES_LIBUTF:%.c=$(BUILDDIR)%.o)
+LIB_SBASE = $(addprefix sbase/,$(LIBUTF)) $(addprefix sbase/,$(LIBUTIL))
+
+PROGRAM = sbase-box
+BINDIR = bin/sbase
+BUILDDIR = build/
+SOURCES = $(SOURCES_SBASE)
```

Comment:
Why do you treat `LIBUTF` and `LIBUTIL` differently than other sources? Wouldn't adding them to `SOURCES` variable simplify this `Makefile`?

Reply:
This was a leftover from the original sbase version of Makefile. I've added libutil and libutf sources to`SOURCES` and simplified the Makefile, so that we don't build `libutil.a` and `libutf.a`.